.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Be careful about adding other code files into your pull request, but I think this will be fine since the team will need this reference, and it created a subfolder for vscode/settings.